### DISCUSSION TITLE
Struggling to configure voice assist, and I can't find the final dialog setting

### DISCUSSION THREAD

**Discussion by mph070770** [music-assistant/support#3090] [Q&A] [answered]
Sorry guys but this is driving me mad.  I've followed the MA voice integration instructions but can't find this setting box:

![image](https://github.com/user-attachments/assets/0352b41e-ba07-4814-a02e-41ca60f304b2)

Please can someone give me instructions on where to find it?  I originally installed music assistant as an addon, so it didn't appear as an integration, so this instruction:

"Navigate to Music Assistant in the Integration view of the HA settings. Click on CONFIGURE in the Integration entries section."

from here: https://music-assistant.io/integration/voice/

didn't seem correct?  I uninstalled the addon and tried adding as a HACS integration and although I've downloaded the package and restarted HA, i'm not getting the option of installing MA - it says the server isn't running :-(

Any help would be much appreciated!

**Comment by OzGav** [comment, marked as answer]
https://music-assistant.io/#architecture

**Reply by mph070770** [reply]
Thanks OzGav - I didn't appreciate the architecture as I had jumped to the voice instructions.  All working now, thanks!

**Comment by kazuyoshifunaki** [comment]
Apologies if it is not correct to "reopen" this. I used to find this menu and also the voice assistant feature worked well with OpenAI. Since the newest version, the configure button is gone. 

- Now my HA Assist doesn't seem to give me proper results anymore (typically, it's "Unexpected error during intent recognition" when I leave the intent files in under custom_sentences. 
- When I remove the intent files, it just doesn't do the correct stuff anymore i.e. when it is already playing something and I give it a command to play something else (e.g. play ABBA on xyz speaker), it just plays the next song of the playlist. If it is stopped, it says the player is paused and it doesn't actually play anything anymore.

I wonder if this is related to the inability to configure the Music Assistance-specific LLM agent. I was looking for this configuration option, but it seems to have disappeared?

PS:
I have to add here that I struggled with the migration and was confused by all the different media_players I suddenly had. I ended up removing all MA instances (add-on, integrations, and also the new instance that was in the HA core), and I just used the newly re-discovered instance from the core.

**Reply by OzGav** [reply]
I assume you are using the HA core version of the Integration? If so, Voice is not supported there as yet.

**Reply by kazuyoshifunaki** [reply]
ah... I thought I was going crazy, was trying for hours to make the Assist feature work again...  any idea, by when this will be supported again?

**Reply by OzGav** [reply]
Maybe next month
